packages/typography/src/Typography.js

```diff
@@ -64,8 +68,15 @@ export default class Typography extends Component {
     return (
       <ThemeContext.Consumer>
         {({ resolvedRoles }) => {
+          const { stylesheet: customStylesheet } = this.props;
```

Comment:
we are already defining `const` from `this.props` above, this should go in that block as well.

Reply:
Done